### PR TITLE
EPSP preparation 1/X

### PR DESCRIPTION
**Pull request by t-b**
@michaelHuth Please review

### CODE REVIEW

File: Packages/MIES/MIES_SweepFormula_Helpers.ipf [hunk 1]
```diff
@@ -12,6 +12,74 @@
 
 static StrConstant SFH_WORKING_DF = "FormulaData"
 
+Function SFH_GetArgumentAsNumeric(variable jsonId, string jsonPath, string graph, string opShort, variable argNum, [variable defValue])
```
Comment (MichaelHuth) [inline Packages/MIES/MIES_SweepFormula_Helpers.ipf:15]: I think some brief function docu would be helpful.
(also for `SFH_GetArgumentAsWave`)

**Review by MichaelHuth — CHANGES_REQUESTED** [review]
See comments.

**Review by MichaelHuth — DISMISSED** [review]
Looks good.

**Comment by t-b** [comment]
@MichaelHuth Thanks for the review. Will merge once CI passes.

**Pull request merged**